codecov.yml

```diff
@@ -0,0 +1,8 @@
+# This file configures CodeCov to require 90% code coverage. New PRs can
```

Comment:
Below it says 80%? I'd remove the percentage from the comment to avoid having to keep them in sync.